lib/molecules/menu/menu.directive.js

```diff
@@ -10,6 +10,8 @@ function lnMMenu($location) {
     templateUrl: 'lnPatterns/molecules/menu/template.html',
     link: link,
     scope: {
+      lnParentClass: '@',
+      lnChildClass: '@',
```

Comment:
I think these parameters should follow the Parent/Item nomenclature, as it is defined for the lnSelParentClass and lnSelItemClass. `lnChildClass` should be changed to `lnItemClass`.